streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -469,6 +469,9 @@ public void setAssignmentMetadata(final Map<TaskId, Set<TopicPartition>> activeT
             }
         }
 
+        log.debug("Assigning metadata with activeTasks: {}. The updated active task states are: \n" +
```

Comment:
Can we include addedStandbyTasks  / assignedStandbyTasks / revokedActiveTasks / revokedStandbyTasks as well?